controls/cis_rhel8.yml

```diff
@@ -1451,51 +1671,91 @@ controls:
     rules:
       - service_rsyslog_enabled
 
+  - id: 4.2.1.3
+    title: Ensure journald is configured to send logs to rsyslog (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    status: automated
+    rules:
+      - journald_forward_to_syslog
+
   # NEEDS RULE
   # The rsyslog_files_permissions rule is not sufficient
   # https://github.com/ComplianceAsCode/content/issues/7332
-  - id: 4.2.1.3
+  - id: 4.2.1.4
     title: Ensure rsyslog default file permissions configured (Automated)
     levels:
       - l1_server
       - l1_workstation
-    automated: no
+    status: pending
 
-  - id: 4.2.1.4
+  - id: 4.2.1.5
     title: Ensure logging is configured (Manual)
     levels:
       - l1_server
       - l1_workstation
     status: manual
 
-  - id: 4.2.1.5
-    title: Ensure rsyslog is configured to send logs to a remote log host (Automated)
+  - id: 4.2.1.6
+    title: Ensure rsyslog is configured to send logs to a remote log host (Manual)
     levels:
       - l1_server
       - l1_workstation
     status: automated
-    rules:
+    related_rules:
       - rsyslog_remote_loghost
 
-  - id: 4.2.1.6
-    title: Ensure remote rsyslog messages are only accepted on designated log hosts. (Manual)
+  # NEEDS RULE
+  - id: 4.2.1.7
+    title: Ensure rsyslog is not configured to recieve logs from a remote client (Automated)
     levels:
-      - l1_server
-      - l1_workstation
+      - l2_server
+      - l2_workstation
+    status: pending
+
+  # NEEDS RULE
+  - id: 4.2.2.1.1
+    title:  Ensure systemd-journal-remote is installed (Manual)
+    levels:
+      - l2_server
+      - l2_workstation
     status: manual
-    related_rules:
-      - rsyslog_nolisten
 
-  - id: 4.2.2.1
-    title: Ensure journald is configured to send logs to rsyslog (Automated)
+  # NEEDS RULE
+  - id: 4.2.2.1.2
+    title:  Ensure systemd-journal-remote is configured (Manual)
     levels:
-      - l1_server
-      - l1_workstation
+      - l2_server
+      - l2_workstation
+    status: manual
+
+  # NEEDS RULE
+  - id: 4.2.2.1.3
+    title:  Ensure systemd-journal-remote is enabled (Manual)
+    levels:
+      - l2_server
+      - l2_workstation
+    status: manual
+
+  # NEEDS RULE
+  - id: 4.2.2.1.4
+    title: Ensure journald is not configured to recieve logs from a remote client (Automated)
+    levels:
+      - l2_server
+      - l2_workstation
+    status: pending
+
+  - id: 4.2.2.2
+    title: Ensure journald service is enabled (Automated)
+    levels:
+      - l2_server
+      - l2_workstation
     status: automated
     rules:
-      - journald_forward_to_syslog
+      - service_systemd-journald_enabled
```

Comment:
`systemd-journald` rule is needed, because `service_systemd-journald_enabled` fails on `package systemd-journald is installed` now.

Reply:
The `systemd-journald` service unit is part of `systemd` package. If the `packagename` is not informed to the template, the `servicename` is used as `packagename` by default. I will update the rule in another PR.